examples/orchestrated-apps/nuxt.config.js

```diff
@@ -0,0 +1,43 @@
+export default {
+  components: true,
+  plugins: [
+    '~/plugins/orchestrated.server'
+  ],
+  hooks: {
+    'vue-renderer': {
+      ssr: {
+        context (renderContext) {
+          // Disable style/script injection based on query params parsed
+          // in plugins/orchestrated.server.js
```

Comment:
This is used to avoid double-inclusion when multiple components are orchestrated into the page